core/modules/qana/PostPlugin.cc

```diff
@@ -75,8 +75,14 @@ class PostPlugin : public QueryPlugin {
 
     int _limit;
     std::shared_ptr<query::OrderByClause> _orderBy;
+
+private:
+	static LOG_LOGGER logger;
 };
```

Comment:
Yes log4cxx is thread-safe. Nevertheless it might consume resources to create a logger per file. It'll be easy to create only one logger per module if needed.